README.md

```diff
@@ -1,2 +1,52 @@
-# testcontainers-java-module-vault
-Testcontainers module for HashiCorp Vault
+# TestContainers Vault Module
+
+Testcontainers module for [Vault](https://github.com/hashicorp/vault). Vault is a tool for managing secrets. More information on Vault [here](https://www.vaultproject.io/).
+
+See [testcontainers.org](https://www.testcontainers.org) for more information about Testcontainers.
+
+<!--[![Build Status](https://travis-ci.org/testcontainers/testcontainers-java.svg?branch=master)](https://travis-ci.org/testcontainers/testcontainers-java)-->
+
+## Usage example
+
+Running Vault in your Junit tests is easily done with an @TestRule such as the following:
+
+```java
+public class SomeTest {
+
+    @ClassRule
+    public static VaultContainer vaultContainer = new VaultContainer<>("vault:0.7.0")
```

Comment:
Maybe we should provide the default constructor with pre-set vault version like we already do for PostgreSQL/MySQL/others?